CHANGELOG.md

```diff
@@ -28,4 +29,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
--   
+-   Layers removed when workflow is finished or exited out of
```

Comment:
I think "Fixed" describes fixes compared to the existing version of the released software. Since there is no existing version, I think we can remove this line.